edit/harvesterhci.io.host/index.vue

```diff
@@ -420,7 +420,7 @@ export default {
       </Tab>
       <Tab name="network" :weight="90" :label="t('harvester.host.tabs.network')">
         <InfoBox class="wrapper">
-          <div class="row warpper">
```

Comment:
This class was dangling?

Reply:
I've removed this, if it wasn't missed originally it seems safer to nuke